worlds/sc2/Client.py

```diff
@@ -259,13 +259,17 @@ def color_code(self, code: str):
 
 class SC2Context(CommonContext):
     command_processor = StarcraftClientProcessor
-    game = "Starcraft 2 Wings of Liberty"
+    game = "Starcraft 2"
```

Comment:
This breaks ability to join older slots. I think we shall ask core devs about that

Reply:
Possibility: Can we try to join "Starcraft 2", and then if that fails we try to join the old name and display a warning?